contracts/TokenFactory.sol

```diff
@@ -397,17 +397,22 @@ contract TokenFactory is Ownable {
     }
 
     /**
-     * @dev Withdraws any remaining USDC fees to the specified address.
-     * @param _to Address to which the remaining fees are withdrawn.
+     * @dev Withdraws any stuck tokens (LP Or USDC) to the specified address.
+     * @param _token Address of the token to be withdrawn.
+     * @param _to Address to which the tokens are withdrawn.
      */
 
-    function withdrawFee(address _to) external onlyOwner {
-        if (_to == address(0)) {
+    function withdraw(address _token, address _to) external onlyOwner {
```

Comment:
I've seen this generally called emergencyWithdraw I would rename it

Reply:
Done